pkg/cmd/cli/cmd/export.go

```diff
@@ -92,9 +94,16 @@ func RunExport(f *clientcmd.Factory, exporter Exporter, in io.Reader, out io.Wri
 	if err != nil {
 		return err
 	}
-	outputVersion, err := kcmdutil.OutputVersion(cmd, clientConfig.GroupVersion)
-	if err != nil {
-		return err
+
+	var outputVersion schema.GroupVersion
+	outputVersionString := kcmdutil.GetFlagString(cmd, "output-version")
```

Comment:
> Because I need someone to tell me what we should be doing here instead

`--output-version` shouldn't exist.  It should be removed and we should chase the ripples.